### PR TITLE
Substrate Companion

### PR DESCRIPTION
**Pull request by Doordashcon**
https://github.com/paritytech/substrate/pull/13880

Cumulus companion: https://github.com/paritytech/cumulus/pull/2514

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2718224

File: runtime/test-runtime/src/lib.rs [hunk 1]
```diff
@@ -463,6 +463,7 @@ impl pallet_vesting::Config for Runtime {
 impl pallet_sudo::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type RuntimeCall = RuntimeCall;
+	type WeightInfo = ();
```
Comment (muharem) [inline runtime/test-runtime/src/lib.rs:466]: we need to run the benches, and use an actual weight for of runtimes.

Comment (Doordashcon) [reply runtime/test-runtime/src/lib.rs:466]: Yeah, can we call a bot to do that like is done on the Substrate repo?

Comment (muharem) [reply runtime/test-runtime/src/lib.rs:466]: yes, but I think it wont work until substrate is not merged
You can copy the weights from substrate. During the release process they will be replaced by actual weights.

Comment (Doordashcon) [reply runtime/test-runtime/src/lib.rs:466]: Thanks, will do that.

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2773698

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2775841

**Review by sam0x17 — APPROVED** [review]

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2798184

File: runtime/rococo/src/lib.rs [hunk 2]
```diff
@@ -946,32 +946,33 @@ impl InstanceFilter<RuntimeCall> for ProxyType {
...
-					RuntimeCall::Democracy(..) |
-						RuntimeCall::Council(..) | RuntimeCall::TechnicalCommittee(..) |
-						RuntimeCall::PhragmenElection(..) |
-						RuntimeCall::Treasury(..) |
-						RuntimeCall::Bounties(..) |
-						RuntimeCall::Tips(..) | RuntimeCall::Utility(..) |
-						RuntimeCall::ChildBounties(..)
-				),
+			ProxyType::Governance => matches!(
+				c,
```
Comment (ggwpez) [inline runtime/rococo/src/lib.rs:950]: Please undo unrelated changes to ease review.

**Review by ggwpez — COMMENTED** [review]
Weights will be updated before the next release.

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2809452

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2813108

**Review by ggwpez — APPROVED** [review]

File: runtime/test-runtime/src/lib.rs (hunk 1 above)
Comment (ggwpez) [reply runtime/test-runtime/src/lib.rs:467]: Done for Rococo/Westend. For the testing runtimes it does not matter.

**Comment by ggwpez** [comment]
bot merge
